osmhelper/osm_helper.py

```diff
@@ -69,25 +84,19 @@ def read_stops_from_file():
             return {}
 
     @staticmethod
-    def get_routes(route_type, bbox, refresh=False):
+    def get_routes(bbox, tags, refresh=False):
         if refresh:
             print "Start with fresh routes"
             routes = {}
         else:
             routes = OsmHelper.read_routes_from_file()
 
         # get the master routes first
-        route_masters = OsmHelper.get_route_masters(route_type, bbox, refresh)
+        route_masters = OsmHelper.get_route_masters(bbox, tags, refresh)
 
         api = overpy.Overpass()
-
-        result = api.query("""
-        <query type="relation">
-            <has-kv k="route" v="%s"/>
-            <bbox-query e="%s" n="%s" s="%s" w="%s"/>
-        </query>
-        <print/>
-        """ % (route_type, str(bbox["e"]), str(bbox["n"]), str(bbox["s"]), str(bbox["w"])))
+        query_str = 'relation(%s)%s;(._;);out body;' % (bbox, tags)
```

Comment:
If I run this with the current fenix config file, I get this:
```
Traceback (most recent call last):
  File "./osm2gtfs.py", line 117, in <module>
    main()
  File "./osm2gtfs.py", line 58, in main
    args.refresh_route, "bus", config['query']['bbox'])
  File "/home/user/vcs/osm2gtfs/osmhelper/osm_helper.py", line 306, in refresh_route
    routes = OsmHelper.get_routes(bbox, tags)
  File "/home/user/vcs/osm2gtfs/osmhelper/osm_helper.py", line 99, in get_routes
    result = api.query(query_str)
  File "/home/user/.local/lib/python2.7/site-packages/overpy/__init__.py", line 107, in query
    msgs=msgs
overpy.exception.OverpassBadRequest: Error: line 1: parse error: Unknown query clause 
Error: line 1: parse error: ')' expected - 'bus' found. 
Error: line 1: parse error: An empty query is not allowed 
Error: line 1: parse error: Unknown type "{" 
Error: line 1: parse error: An empty query is not allowed 
Error: line 1: parse error: Unknown type "u"
...
```
Please test your PRs so that the old functionality continues to work. Maybe it would be a good idea to set up a CI for that.